app/src/main/java/com/teamcatchme/catchme/presentation/home/HomeActivity.kt

```diff
@@ -116,6 +107,40 @@ class HomeActivity : BindingActivity<ActivityHomeBinding>(R.layout.activity_home
         }
     }
 
+    private fun serverBinding() {
+        mainApi.main().enqueue(object :
+            Callback<ResponseWrapper<List<ResponseMain>>> {
+            override fun onResponse(
+                call: Call<ResponseWrapper<List<ResponseMain>>>,
+                response: Response<ResponseWrapper<List<ResponseMain>>>
+            ) {
+                if (response.isSuccessful) {
+                    if (response.body()?.data?.size == 0) {
+                        binding.vpHomeCatchu.isVisible = false
+                        binding.indicatorHomeVp.isVisible = false
+                        binding.txtHomeName.isVisible = false
+                        binding.btnHomeCatching.isVisible = false
+
+                        binding.imgEmptyCatchu.isVisible = true
+                        binding.txtEmptyTitle.isVisible = true
+                        binding.txtEmptyDescription.isVisible = true
+                        binding.btnEmptyCatchme.isVisible = true
+
+                    } else {
+                        homeList.clear()
+                        response.body()!!.data.forEach { homeList.add(it.toHomeItem()) }
+                        homeAdapter.submitList(homeList)
```

Comment:
```suggestion
                        val isListEmpty = (response.body()?.data?.size == 0)
                        
                        binding.vpHomeCatchu.isVisible = isListEmpty
                        binding.indicatorHomeVp.isVisible = isListEmpty
                        binding.txtHomeName.isVisible = isListEmpty
                        binding.btnHomeCatching.isVisible = isListEmpty
                        
                        binding.imgEmptyCatchu.isVisible =  !isListEmpty
                        binding.txtEmptyTitle.isVisible =  !isListEmpty
                        binding.txtEmptyDescription.isVisible =  !isListEmpty
                        binding.btnEmptyCatchme.isVisible =  !isListEmpty
                        
                        homeList.clear()
                        response.body()!!.data.forEach { homeList.add(it.toHomeItem()) }
                        homeAdapter.submitList(homeList)
```

이런식으로 visible을 data size가 0이 아닐때에도 visibility 설정해줘야할 것 같아, Restart에서 돌아왔을 때 data size가 0 이상인데 이전에 0이었을 경우에 visibility를 유지하게 된다면 구현 의도와는 달라지겠지?